detect_variants/detect_variants.cwl

```diff
@@ -263,11 +274,30 @@ steps:
             vcf: bgzip/bgzipped_file
         out:
             [indexed_vcf]
+    annotated_vcf_filter:
```

Comment:
`cle_filtered_vcf` this is or is not filtered depending on the `cle_filter` boolean flag.